templates/focus-ios.xliff

```diff
@@ -217,6 +258,10 @@
         <source>Cancel</source>
         <note>Label on button to cancel edits</note>
       </trans-unit>
+      <trans-unit id="Disabled">
+        <source>Disabled</source>
+        <note>label describing something as disabled</note>
```

Comment:
Where are you using this? Most languages need to adapt adjectives to the name's gender, and this feels like a catch-all label, which is wrong.